app/src/pages/inside/stepPage/modals/editDefectModals/makeDecisionModal/makeDecisionModal.jsx

```diff
@@ -266,11 +316,101 @@ const MakeDecision = ({ data }) => {
           } else {
             setIssueAction({ actionName: UNLINK_ISSUE, nextAction: handleUnlinkIssue });
           }
+          !isEmptyObject(copyFromHistoryState) && setCopyFromHistoryState({});
+          collapseTabsExceptCurr(TABS_ID[SELECT_DEFECT_MANUALLY]);
         },
+        isSelected: issueAction.actionName === UNLINK_ISSUE,
       });
     }
     return actionButtonItems;
   };
+  const preparedHistoryLineItems = historyItems.filter(
+    (item) => item.issue && item.id !== itemData.id,
+  );
+  const selectHistoryLineItem = (itemId) => {
+    if (itemId) {
+      const historyItem = preparedHistoryLineItems.find((item) => item.id === itemId);
+      setCopyFromHistoryState(historyItem);
+      setSelectDefectTypeState({ issue: itemData.issue });
+      collapseTabsExceptCurr(TABS_ID[COPY_FROM_HISTORY_LINE]);
+    } else {
+      setCopyFromHistoryState({});
+    }
+    setIssueAction({});
```

Comment:
Why we need to reset  issue action  in this case?

Reply:
issue action is a part of select defect type manually tab, a user can press link issue and go to copy from history line tab and select something there, so I'm need reset issue actions

Reply:
Got it.